moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -74,6 +74,7 @@ class Servo
   /**
    * \brief Computes the joint state required to follow the given command.
    * @param command The command to follow, std::variant type, can handle JointJog, Twist and Pose.
+   * is used to calculate an error between the current state and the previous commanded state for anti-drift correction.
```

Comment:
Continuing the formatting nitpick theme  ;)

```suggestion
   * Is used to calculate an error between the current state and the previous commanded state for anti-drift correction.
```